CMakeLists.txt

```diff
@@ -59,6 +59,11 @@ if(BUILD_ASAN)
   target_compile_options(${LRS_TARGET} PRIVATE -fsanitize=address -g)
 endif()
 
+# Add extra security flags that are mandatory in the debian build to fail fast. See RSDSO-20629.
+if(LINUX)
```

Comment:
Maybe better to add it to our security flags section?

See 
https://github.com/IntelRealSense/librealsense/blob/cacc8e701a963a44b49fe51911443797e8b3a517/CMake/unix_config.cmake#L51

Enabled on GHA flow using 
-DENABLE_SECURITY_FLAGS

This way we don't change how it will be built for users but add us a level of protection

Reply:
Done